src/components/Button/_button.scss

```diff
@@ -26,97 +26,93 @@
 .btn--secondary,
 .btn--ghost {
   @extend %btn-size-standard;
-  padding: 0 calc(2.5 * var(--space-unit));
-}
-
-.btn--primary {
-  background-color: var(--link-colour);
-  color: var(--colour-white);
-
-  &:hover {
-    background-color: var(--link-colour-hover);
-    color: var(--colour-white);
-  }
 
-  &:focus {
-    background-color: var(--link-colour);
-    color: var(--colour-white);
-  }
+  color: var(--colour-grey-80);
+  padding: var(--space-unit) calc(2 * var(--space-unit));
 
+  &:hover,
+  &:focus,
   &:active {
-    background-color: var(--link-colour-active);
-    color: var(--colour-white);
+    color: var(--colour-grey-80);
   }
+}
 
+.btn--primary,
+.btn--secondary {
   &[disabled] {
-    background-color: var(--colour-grey-20);
+    background-color: var(--colour-grey-10);
   }
 }
 
-.btn--secondary,
-.btn--ghost,
-.btn--link {
-  color: var(--link-colour);
+.btn--primary {
+  background-color: var(--colour-amber-30);
 
   &:hover {
-    color: var(--link-colour-hover);
+    background-color: var(--colour-amber-20);
   }
 
+  &:focus,
   &:active {
-    color: var(--link-colour-active);
+    background-color: var(--colour-amber-30);
   }
 }
 
-.btn--secondary,
-.btn--ghost {
-  border: 1px solid var(--link-colour-active);
+.btn--secondary {
+  background-color: var(--colour-amber-20);
 
   &:hover {
-    border-color: var(--link-colour);
+    background-color: var(--colour-amber-10);
   }
 
+  &:focus,
   &:active {
-    border-color: var(--link-colour-active);
+    background-color: var(--colour-amber-20);
   }
+}
 
-  &[disabled] {
-    border-color: var(--link-colour-disabled);
+.btn--ghost {
+  background-color: transparent;
+
+  &:hover {
+    background-color: var(--colour-amber-20);
   }
-}
 
-.btn--secondary {
-  background-color: var(--colour-white);
+  &:focus,
+  &[disabled] {
+    background-color: transparent;
+  }
 
   &:active {
-    background-color: var(--colour-blue-10);
+    background-color: var(--colour-amber-10);
   }
 
-  &[disabled] {
-    background-color: var(--colour-grey-20);
+  .btn__text {
```

Comment:
This child `.btn__text` and the following state selectors should be flattened as it's becoming harder to read here

i.e. 

```
.btn--tertiary .btn__text {
  ...
}

.btn--tertiary:hover .btn__text,
.btn--tertiary:active .btn__text,
.btn--tertiary[disabled] .btn__text {
  ...
}
```